test/integration/README.md

```diff
@@ -0,0 +1,19 @@
+# CRC integration tests
+
+CRC integration tests use Clicumber package which provides basic functionality for testing CLI binaries.
+Clicumber allows to run commands in persistent shell instance (shell, tcsh, zsh, cmd, or powershell), assert its outputs (stdout, stderr or exitcode) and also allows to check configuration files etc.
+
+The general functionality of Clicumber is then extended by CRC specific test code to cover the whole functionality of CRC.
+
+## How to run
+
+1. get the Clicumber package: `go get -u github.com/agajdosi/clicumber`
```

Comment:
We agree on using `github.com/agajdosi/clicumber` then I will suggest that we move it to `code-ready` namespace in github.

Reply:
Ok.

Reply:
Since we are not able to make changes in clicumber under @agajdosi's personal repository. I opt for creating the new repository for it under `code-ready` organization before we proceed with integration tests for crc.